source/installation/install-docker.txt

```diff
@@ -0,0 +1,134 @@
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. This deployment method is ideal for 
+educational purposes. Use this if you want to learn how to configure your own 
+multi-server Kafka environment. 
+
+.. warning::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+- Relational Migrator is available on port 8080.
+
+- In the Docker container, Relational Migrator is installed to 
+  ``/opt/mongodb-relational-migrator/``.
+
+- When running in Docker, the ``user.properties`` file is ignored. Define any 
+  property overrides using environment variables. For example: 
+
+  .. code-block::
+
+     docker run -p 8080:8080 -e MIGRATOR_APP_TELEMETRY_ENABLE=false public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+- `Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
+  persisting data and/or drivers if a container stops running or is restarted. 
+  You can create volumes to the paths:
+
+  - For data and logs: ``/root/Migrator``
+  - For drivers: ``/opt/mongodb-relational-migrator/lib/app/lib``
+
+  The following example runs Relational Migrator with persistent volumes for drivers 
+  and data:
+
+  .. code-block::
+
+     docker run -p 8080:8080 -v /migrator/drivers:/opt/mongodb-relational-migrator/lib/app/lib -v\
+     /migrator/data:/root/Migrator  public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+Before you Begin
+----------------
+
+You must have Docker installed on your computer. For more information, see  
+`Install Docker Engine <https://docs.docker.com/engine/install/>`__. 
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install a Kafka cluster
+
+      Use the following ``docker-compose`` configuration as an example to install a new 
+      Kafka cluster, or connect to an existing cluster.
+
+      .. code-block::
+
+         version: '3'
+         services:
+           kafka-1:
+           image: quay.io/debezium/kafka:2.2
+           ports:
+             - 19092:9092
+             - 19093:9093
+           environment:
+             CLUSTER_ID: oh-sxaDRTcyAr6pFRbXyzA
+             BROKER_ID: 1
+             KAFKA_CONTROLLER_QUORUM_VOTERS: 1@kafka-1:9093
+             NODE_ROLE: controller,broker
+           healthcheck:
+             test: ["CMD-SHELL", "./bin/kafka-topics.sh --version"]
+             interval: 5s
+             timeout: 10s
+             retries: 10
+
+   .. step:: Install a Kafka Connect cluster
+
+      Install a Kafka Connect cluster with the ``relational-migrator-connect`` image 
+      from the `public Amazon ECR <public.ecr.aws/v4d7k6c9/relational-migrator:latest>`__.
```

Comment:
This is should not be a hyperlink; we need to show the full path of the container in the docs.

Reply:
This link does not work for me